### DISCUSSION TITLE
Craft 5 - Add handle to UID for field content

### DISCUSSION THREAD

**Discussion by engram-design** [craftcms/cms#13805] [Ideas] [answered]
The new content column is pretty great, producing:

```
{
	"93228598-1817-46bc-80a4-8fcc2054b3a1": "Text 1",
	"94c4446d-7581-473d-81d8-0b2cda6381a0": "Text 2"
}
```

Which are the UIDs for fields. However, I'd like to propose including the handle of fields as well. This will not only help the transition, but it's pretty handy when needing to manually alter this content. It's happened more times than we'd care to admit in an emergency.

```
{
	"93228598-1817-46bc-80a4-8fcc2054b3a1:aboutUs": "Text 1",
	"94c4446d-7581-473d-81d8-0b2cda6381a0:tagline": "Text 2"
}
```

**Comment by brandonkelly** [comment, marked as answer]
This would require keys to be kept up-to-date with field handles (globally or within the field layout’s Handle overrides), as the JSON keys need to be explicitly referenced for field value expressions (see `Field::getValueSql()`). Which would make renaming field handles a seriously expensive operation on larger sites, so not worth the DX convenience.

**Reply by engram-design** [reply]
Ah, gotcha. Makes sense then!
